### PR TITLE
hotfix(ci.jenkins.io) correct agent path by factorizing to hieradata

### PR DESCRIPTION
**Pull request by dduportal**
Signed-off-by: Damien Duportal <damien.duportal@gmail.com>

### CODE REVIEW

File: dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb [hunk 1]
```diff
@@ -212,12 +212,17 @@ jenkins:
                   key: "JENKINS_JAVA_BIN"
                   value: "<%= agent['agentJavaBin'] ? agent['agentJavaBin'] : @jcasc['agents_setup']['ubuntu']['agentJavaBin'] %>"
             <%- if agent['javaHome'] -%>
+            <>
```
Comment (lemeurherve) [inline dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb:215]:
```suggestion
```
typo?

File: hieradata/common.yaml [hunk 2]
```diff
@@ -213,6 +213,7 @@ profile::jenkinscontroller::jcasc:
       osDiskSize: 90
       agentJavaBin: '/opt/jdk-11/bin/java'
       agentJavaOpts: '-XX:+PrintCommandLineFlags'
+      path: '~/.asdf/shims:~/.asdf/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin'
```
Comment (lemeurherve) [inline hieradata/common.yaml:216]: /usr/**games**? ^^

Comment (dduportal) [reply hieradata/common.yaml:216]: Yes :D It is the default Ubuntu 20.04 PATH (extracted in https://ci.jenkins.io/job/Infra/job/acceptance-tests/job/check-agent-availability/1736/console and confirmed on a local VM)

File: dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb (hunk 1 above)
Comment (dduportal) [reply dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb:215]: 👍 
fixed post-merge in https://github.com/jenkins-infra/jenkins-infra/commit/789d922c9f7396b523b1f2b4b10087f25c46126e

**Pull request merged**